test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -84,13 +88,40 @@ protected Logger getResultsLogger() {
 						multipleAutoAccountCreations(),
 						accountCreatedIfAliasUsedAsPubKey(),
 						aliasCanBeUsedOnManyAccountsNotAsAlias(),
-						autoAccountCreationWorksWhenUsingAliasOfDeletedAccount()
+						autoAccountCreationWorksWhenUsingAliasOfDeletedAccount(),
+						canGetBalanceAndInfoViaAlias(),
 				}
 		);
 	}
 
+	private HapiApiSpec canGetBalanceAndInfoViaAlias() {
+		final var ed25519SourceKey = "ed25519Alias";
+		final var secp256k1SourceKey = "secp256k1Alias";
+		final var secp256k1Shape = KeyShape.SECP256K1;
+		final var ed25519Shape = KeyShape.ED25519;
+		final var autoCreation = "autoCreation";
+
+		return defaultHapiSpec("CanGetBalanceAndInfoViaAlias")
+				.given(
+						newKeyNamed(ed25519SourceKey).shape(ed25519Shape),
+						newKeyNamed(secp256k1SourceKey).shape(secp256k1Shape)
+				).when(
+						cryptoTransfer(
+								tinyBarsFromAccountToAlias(GENESIS, ed25519SourceKey, ONE_HUNDRED_HBARS),
+								tinyBarsFromAccountToAlias(GENESIS, secp256k1SourceKey, ONE_HUNDRED_HBARS)
+						)
+								.payingWith(GENESIS)
+								.via(autoCreation)
+				).then(
+						getAliasedAccountBalance(ed25519SourceKey),
+						getAliasedAccountBalance(secp256k1SourceKey),
+						getAliasedAccountInfo(ed25519SourceKey).hasExpectedAliasKey(),
+						getAliasedAccountInfo(secp256k1SourceKey).hasExpectedAliasKey()
```

Comment:
Used `getAccountInfoWithAlias` and `getAccountBalanceWithAlias` for other test suites. Will clean up tomorrow in a different PR.